kill-all.sh

```diff
@@ -26,6 +28,23 @@ while getopts ':hg:p:m:' option; do
   esac
 done
 
+docker exec $PROMETHEUS_NAME kill 1
+TRIES=0
+OK=0
+until [ $OK -eq 1 ] || [ $TRIES -eq 10 ]; do
+    if VAL=`docker logs aprom|&tail -1 |grep 'See you next time'`; then
```

Comment:
we exit the loop if the command fail, if we see that prometheus completed successfuly or if we run out of retries.

There should be a point we give up on waiting. I think 10s is more than enough, we can set it to anyother number, but it should be limited by something.

Worst thing, it will not be a graceful shutdown.